test/is-empty/index.js

```diff
@@ -0,0 +1,23 @@
+var test = require('tape');
```

Comment:
please also add tests for 1) empty array, 2) non empty array, 3) undefined and 4) no argument passed